app/app.go

```diff
@@ -214,7 +214,7 @@ type App struct {
 
 	LaunchKeeper launchmodulekeeper.Keeper
 
-	AccountKeeper accountmodulekeeper.Keeper
+	AccountKeeper profilemodulekeeper.Keeper
```

Comment:
```suggestion
	ProfileKeeper profilemodulekeeper.Keeper
```

We will also need to specify this change inside `x/launch/types/expected_keeper`